packages/vue/src/components/FInteractiveTable/FInteractiveTable.vue

```diff
@@ -616,6 +618,49 @@ export default defineComponent({
              */
             this.$emit("update:active", this.activeRow);
         },
+        confirmUniqueKey(): void {
+            const seenKeys: Record<string, boolean> = {};
```

Comment:
Ett annat snällare sätt (men kan kräva att vi tänker lite) är om vi hade genererat ett unikt internt id istället så konsumenten inte måste sätta `key-attribute` och garantera att de har en unik identifierare. Så använder vi bara det interna id:t och konsumenten får själv lösa om de har duplicates bland sina värden.

Reply:
Japp, det känns som att i det läget är rekommendationen att då sätta `key-attribute`. Uppdateringar är fine men inte att byta ut hela objektet (eller ersätta med en nya array med nya objekt).

Reply:
Senast just nu blev jag bränd av just detta :sob: Jag har en pinia store där jag inte har någon naturlig primärnyckel, här är inte lika enkelt att använda ett löpnummer eftersom jag inte direkt äger arrayen, jag antar att jag får peta in ett "id" vid insertion som är `max(...ids) + 1` eller något och sen måste det ju då sparas i min store. Kanske fula det med att skapa en konkaternering av alla kolumner?